retailer-oas/retailer.yaml

```diff
@@ -459,63 +714,256 @@ components:
     GeoCoordinates:
       type: object
       properties:
-        lat: 
+        lat:
           type: number
           description: Latitude in decimal degrees
-        lon: 
+        lon:
           type: number
           description: Longitude in decimal degrees
 
     Capacity:
       type: object
       properties:
-        id:
-          type: string
-        total: 
+        total:
           type: number
+          description: Total capacity of the location
+          example: 100
         occupied:
           type: number
+          description: Occupied capacity of the location
+          example: 20
 
     TimeSlot:
       type: object
       properties:
         id:
           type: string
+          description: Unique identifier of the time slot
+          example: "slot1"
         fromDate:
           type: string
           format: ISO 8601
+          description: The starting time of the timeslot
+          example: "2023-01-11T14:00:00+00:00"
         toDate:
           type: string
           format: ISO 8601
-        
-    SlotAvailability:
+          description: The ending time of the timeslot
+          example: "2023-01-11T15:00:00+00:00"
+
+    TimeSlotAvailability:
+      type: array
+      items:
+        type: object
+        properties:
+          timeSlot:
+            $ref: '#/components/schemas/TimeSlot'
+          capacity:
+            $ref: '#/components/schemas/Capacity'
+          available:
+            type: boolean
+            description: Indicates if the user can select this timeslot. Usually
+            if still have free capacity.
+            example: true
+
+    Schedule:
       type: object
       properties:
         id:
           type: string
-        # TODO: storeId? locationCode? pickingLocationId? placeId? coords?
-        # does the availability varies dependending on the kind of services? pick vs deliver vs click&collect vs ...
-        locationId: 
-          type: string
-        # date:
-        #   type: string
-        #   format: ISO 8601
-        timeSlot:
-          $ref: '#/components/schemas/TimeSlot'
-        capacity:
-          $ref: '#/components/schemas/Capacity'
-        available:
-          type: boolean
+        slots:
+          type: array
+          items:
+            $ref: '#/components/schemas/TimeSlotAvailability'
 
-    Schedule:
+    OrderArticle:
```

Comment:
Let's discuss naming here. We should define the differences between Product and Article/Item or w/e name we decide.